tests/scripts/all.sh

```diff
@@ -1185,8 +1185,8 @@ component_test_platform_calloc_macro () {
 
 component_test_malloc_0_null () {
     msg "build: malloc(0) returns NULL (ASan+UBSan build)"
-    scripts/config.pl full
-    scripts/config.pl unset MBEDTLS_MEMORY_BUFFER_ALLOC_C
+    scripts/config.py full
+    scripts/config.py unset MBEDTLS_MEMORY_BUFFER_ALLOC_C
```

Comment:
This line is actually no longer necessary since `full` no longer sets `MBEDTLS_MEMORY_BUFFER_ALLOC_C`. But that's a separate issue and the line is harmless. We'll fix it later.